cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -134,6 +134,15 @@ for k,v in yaml.load(sys.stdin).iteritems():
 ''' < "${kube_env_yaml}")
 fi
 
+# Wait for the default service account
+token_found=""
+while [ -z "${token_found}" ]; do
```

Comment:
I considered this for a short while, but there's almost no recourse on timeout. You can declare defeat, but you'll have no add-on objects created, or worse, you'll create them badly.